src/api/analytics/gtag.js

```diff
@@ -0,0 +1,192 @@
+import { links, options } from 'config'
+import formatFacetNames from 'helpers/formatFacetNames'
+import formatFilterName from 'helpers/formatFilterName'
+
+const { transformationHumanReadable: transformation } = options
+
+// adds a custom event to GA4
+// https://developers.google.com/analytics/devguides/collection/ga4/events?client_type=gtag
+const event = (eventName, value = {}, nonInteraction = false) => {
+  window.gtag('event', eventName, {
+    ...value,
+    non_interaction: nonInteraction
+  })
+}
+
+/* --- General --- */
+const emailSubscription = (from) =>
+  event('email_subscription', { subscription_click_from: from })
+
+/* --- Compendia --- */
+const compendiaDownload = (type, organism) =>
+  event(`compendia_${type}_download`, {
+    [`${type}_organism`]: organism
+  })
+
+/* --- Datasets --- */
+const myDatasetAction = (action) =>
+  event('my_dataset_action', {
+    my_dataset_action: `${action[0].toUpperCase()}${action.substring(
+      1
+    )} Samples`
+  })
+
+const myDatasetDownloadOptions = (dataset) =>
+  event('my_dataset_download_options', {
+    my_dataset_download_options: `Aggregate: ${
+      dataset.aggregate_by
+    }, Transformation: ${transformation[dataset.scale_by]}, QN: ${
+      dataset.quantile_normalize ? 'Skipped' : 'Not skipped'
+    }
+  `
+  })
+
+const datasetDownload = (token) => event('dataset_downalod', { token })
```

Comment:
```suggestion
const datasetDownload = (token) => event('dataset_download', { token })
```

Which token does this refer to?